docs/resources/cluster.md

```diff
@@ -91,9 +91,9 @@ Property | Description | For Bundles | Default
 `password_authentication`|Accepts true/false. Require clients to provide credentials — a username & API Key — to connect to the Spark Jobserver.|Spark|false
 `target_kafka_cluster_id`|GUID of the Instaclustr managed Kafka Cluster Id you wish to connect to. Must be in the same Instaclustr account.|Kafka Connect|Required
 `vpc_type`|Available options: `KAFKA_VPC`, `VPC_PEERED`, `SEPARATE_VPC`. Only required if targeting an Instaclustr managed cluster.|Kafka Connect|`SEPARATE_VPC`
-`aws_access_key`, `aws_secret_key`, `s3_bucket_name`|Access information for the S3 bucket where you will store your custom connectors. (if using AWS)|Kafka Connect
-`azure_storage_account_name`, `azure_storage_account_key`, `azure_storage_container_name`|Access information for the Azure Storage container where you will store your custom connectors.|Kafka Connect
-`ssl_enabled_protocols`, `ssl_truststore_password`, `ssl_protocol`, `security_protocol`, `sasl_mechanism`, `sasl_jaas_config`, `bootstrap_servers`|Connection information for your Kafka Cluster. These options are analogous to the similarly named options that you would place in your Kafka Connect worker.properties file. Only required if connecting to a Non-Instaclustr managed Kafka Cluster|Kafka Connect
+`aws_access_key` (deprecated), `aws_secret_key` (deprecated), `s3_bucket_name`|Access information for the S3 bucket where you will store your custom connectors. (if using AWS)|Kafka Connect
```

Comment:
so if they're deprecated, what have they been replaced with, and is the replacement mentioned anywhere?

Reply:
The replacement is below, there's an additional block of `kafka_connect_credential` that takes care of this. I'll add `see kafka_connect_credential` to make it clearer